pelicanconf.py

```diff
@@ -11,4 +11,8 @@
 from config.pelicanconf_flags import *
 
 SITEURL = os.getenv("SITEURL")
+
+if not SITEURL:
+    raise ValueError("SITEURL environment variable is not set")
+
```

Comment:
El hecho dec hacer este check es que me he pegado un rato con un error `None has not endswith method` y sólo por el coraje que me ha dadohe dicho: que arda todo!